core/templates/components/question-directives/question-player/question-player.component.ts

```diff
@@ -338,7 +338,7 @@ angular.module('oppia').component('questionPlayer', {
         createScorePerSkillMapping();
         $scope.resultsLoaded = false;
         var totalQuestions = Object.keys(questionStateData).length;
-        for (var question in questionStateData) {
+        questionStateData.forEach(question => {
```

Comment:
Just to check, `questionStateData` is a dict. Can we use `forEach` by itself over a dict?

Reply:
@aks681 Changed for in loop to for of loop as it works for dict